src/ray/core_worker/core_worker.cc

```diff
@@ -270,7 +270,8 @@ void CoreWorkerProcess::RunTaskExecutionLoop() {
   } else {
     std::vector<std::thread> worker_threads;
     for (int i = 0; i < instance_->options_.num_workers; i++) {
-      worker_threads.emplace_back([]() {
+      worker_threads.emplace_back([i] {
+        SetThreadName("worker.task" + std::to_string(i));
```

Comment:
Just a question; Threads names don't need to globally unique right? Is it identified as (pid, thread_name)?

Reply:
- Yes, thread name don't need to globally unique, but sometimes I just want to see if these `worker.task` are load-balance.